doc/whats_new.rst

```diff
@@ -97,6 +97,9 @@ BUG
 
     - Add copy parameter in :func:`mne.Epochs.apply_baseline` method by `Jona Sassenhagen`_ and `Alex Gramfort`_
 
+    - Fig bug in :func:`mne.merge_events` when using ``replace_events=False`` by `Alex Gramfort`_
```

Comment:
Please do

Thx

> On 27 mars 2016, at 21:46, Teon L Brooks <notifications@github.com> wrote:
> 
> In doc/whats_new.rst:
> 
> > @@ -97,6 +97,9 @@ BUG
> >  
> >      - Add copy parameter in :func:`mne.Epochs.apply_baseline` method by `Jona Sassenhagen`_ and `Alex Gramfort`_
> >  
> > +    - Fig bug in :func:`mne.merge_events` when using ``replace_events=False`` by `Alex Gramfort`_
> realized that it would be just easier for me to correct it
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
>